src/ecmech/ECMech_slipgeom.h

```diff
@@ -99,20 +99,18 @@ namespace ecmech {
             fillFromMS(this->_P_ref_vec, this->_Q_ref_vec,
                        mVecs, sVecs, this->nslip);
 
-            int iParam = parsIt - params.begin();
-            assert(iParam == nParams);
+            // assert((parsIt - params.begin()) == nParams);
```

Comment:
Why did this one and some of the others in this file get commented-out?

Reply:
At least for these ones, it was because there were no inputs. So, the check was unnecessary. I think I'd planned on removing these before the PR was pulled in but just commented them out for now for review process.

Reply:
For the review, please put them into whatever final condition is appropriate. Stray "dead code" can be confusing. I would be tempted to leave them in and put them in ifdefs like the others just to keep things consistent, in case some of the functions end up pulling parameters in later modifications, or getting used as examples when people are writing alternative implementations.